apps/docs/README.md

```diff
@@ -0,0 +1,30 @@
+# `docs`
+
+**docs** is a private package for the Fuel documentation site.
+
+## Table of contents
+
+- [`docs`](#docs)
+- [Table of contents](#table-of-contents)
+  - [Building](#building)
+  - [Testing](#testing)
+
+## Building
+
+The build process will build out our snippets and the documentation.
+
+```sh
+pnpm build
+```
+
+The snippets are built into testable scripts from the `src/snippets` folder and generates the Typegen types into the `src/snippets/typegend` folder. All test scripts end with a `.test.ts` suffix.
+
+## Testing
+
+This will build the snippets and run the generated tests. To test a specific environment (`node` or `browser`), the snippet should be named as `{name}.{environment}.test.ts`. e.g. `deploy-contract.node.test.ts`
```

Comment:
We had [discussed this before ](https://github.com/FuelLabs/fuels-ts/pull/3297/files/088d76810196f7dc38cd3d086ff1163c5444d124#r1804062940)on a previous PR but essentially @danielbate made the point about us standardizing how we want to specify test environments by continuing to use the _group_ comments as opposed to using file suffixes. I think we should perhaps continue in that vain.

Reply:
6b0b25fac334db850d032d28cd5b713438f9cb3f